src/beacon.js

```diff
@@ -73,6 +73,7 @@ export default function Emitter(endpoint, context) {
         }
       });
     } else {
+      // eslint-disable-next-line no-constant-condition
```

Comment:
🤔

Reply:
You can argue for changing the approach on this -- it escapes with a break.